### DISCUSSION TITLE
Does GLSP plan to support nls?

### DISCUSSION THREAD

**Discussion by wss29** [eclipse-glsp/glsp#932] [Q&A] [answered]
Hi,
    you know glsp has a menu Diaragm which is English language, If we want to change the language use nls, is it possible? I look at glsp-layout-commands.ts file, it seems that glsp use hardcode label currently, will glsp plan to support multi language use nls 
 like 'nls.localize('label', 'label')'?
![image](https://user-images.githubusercontent.com/4537809/221352852-78be4b7c-330a-402c-8768-77beb84dcd2b.png)
![image](https://user-images.githubusercontent.com/4537809/221352806-491cdf99-3b64-4199-9d8c-7525c5db2a98.png)

**Comment by planger** [comment, marked as answer]
Hi,

good idea, I've created https://github.com/eclipse-glsp/glsp/issues/935. We currently don't have it on our roadmap, but we'd be happy to receive a pull request making those labels translatable.
